actionview/lib/action_view/helpers/tags/collection_radio_buttons.rb

```diff
@@ -14,7 +14,8 @@ def radio_button(extra_html_options={})
         end
 
         def render(&block)
-          render_collection do |item, value, text, default_html_options|
+          options = @options.stringify_keys
```

Comment:
I think we can simplify the implementation of this method and collection_check_boxes moving it `render_collection`. Also I don't think we need to stringify in this case.